README.md

```diff
@@ -10,7 +10,7 @@ A **self-regulating** community is a community in which the users themselves col
 
 Virtually all major internet platforms today do not fit the description above. Take Facebook as an example community. The Facebook community is the set of Facebook users who all understand that every Facebook user is also part of the Facebook community. The Facebook community interacts through Facebook posts, messages, comments, etc. The community rules are the Facebook content policy and Facebook, Inc is ultimately responsible for enforcing them. Thus Facebook is a community as defined above, yet it is not self-hosted since one must rely on Facebook servers to access the community. It is also not self-regulating since Facebook, Inc unilaterally sets the content policy and unilaterally enforces it, without any mechanism for Facebook users to have input on the rules or their enforcement. This is a problem since the incentives of the central operator can become grossly misaligned with the welfare of the community without an effective way for the community to respond.
 
-A possible reaction to the current centralized Internet platforms might be to build a completely decentralized, censorship-resistant, anonymous social media platform. Such a platform certainly doesn't suffer from a central authority abusing its power, however it is not self-regulating since there is no way to enforce any rules on such a system. In fact it barely fits the definition of a community above. Such a system can easily devolve into chaotic, uncivil pandemonium of the sort we see in less-scrupulously moderated social media platforms already existing today.
```

Comment:
I thought the wording became slightly strong, I like keeping it neutral which was well done with the facebook example

Reply:
I agree the wording in this passage is strong, but I do feel that such a platform would not actually be a community.

Maybe we should reword "easily devolve into chaotic, uncivil pandemonium" to tone it down while still conveying the same point.

Reply:
I think that part is fine, my main concern was `In fact it barely` in conjunction with the above. The `In fact it barely` feels like 2 strong terms `in fact` and `it barely`. I'm fine with my currently proposed suggestion of `it does not necessarily` but open to other suggestions if the original meaning is lost

Reply:
Sure, my only thinking here is that I want to convey that this system is not a community unambiguosly.

I'm good with: `In fact it does not`